Blog: draft announcement for setup instructions being translatable

cassidyjames opened the pull request:
@razzeee shared this in the Matrix channel but is unsure if we should share it more widely yet—regardless, I'm getting it drafted here for if/when we do want to share it.

razzeee commented:
1. Weblate fails to merge the translations pretty regularly now - not sure why
2. https://github.com/flathub-infra/website/issues/4137 might be nice to incorporate
3. We should work on having the english translations exported to flatpak

bbhtt committed:
Blog: draft announcement for setup instructions being translatable

@razzeee shared this in the Matrix channel but is unsure if we should share it more widely yet—regardless, I'm getting it drafted here for if/when we do want to share it.

cassidyjames committed:
Merge branch 'main' into blog-setup-translatable
